README.md

```diff
@@ -98,6 +98,42 @@ function visible() {
 
 When your element is near `300px` of the viewport, you get your callback / true result.
 
+### Specifying debounce value
+
+By default, debounce value for scroll and resize events is 15ms.
+
+You can change that with `debounce` param.
+
+```js
+var inViewport = require('in-viewport');
+var elem = document.getElementById('myFancyDiv');
+
+inViewport(elem, { debounce: 300 }, visible);
+
+function visible() {
+  alert('myfancyDiv is visible in the `customContainer`!');
+}
+```
+
+### Failsafe check
+
+By default, inViewport does a failsafe to handle display manipulation that does not throw an event. It works with a `setInterval` performed every 150ms.
+
+Usecase: a hidden parent containing elements; when the parent becomes visible, we have no event that the children became visible.
+
+If you don’t want to run that check constantly, turn it off with `failsafe` param.
```

Comment:
"If you don’t want to run that check constantly"

As a user I will ask myself: "Why?". There's no precise answer on facts here so it raises [FUD](https://en.wikipedia.org/wiki/Fear,_uncertainty_and_doubt)